src/text/atlas.rs

```diff
@@ -184,4 +190,8 @@ impl Atlas {
             );
         }
     }
+    fn delete(&mut self) {
+        let ctx = &mut get_context().quad_context;
+        ctx.delete_texture(self.texture);
```

Comment:
I believe we used to have `delete` on everything, but now, with Drop, should it really be a separate function, not just `drop`?

Reply:
Yeah that's fair, it's probably an entirely unnecessary convention to follow now, I'll remove the delete function and just have the Drop impl :)